templates/plugin/cpu.conf.erb

```diff
@@ -6,5 +6,9 @@
 <% if scope.lookupvar('collectd::collectd_version_real') and scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '5.6']) >= 0 -%>
   ReportNumCpu <%= @reportnumcpu %>
 <% end -%>
+<% if scope.lookupvar('collectd::collectd_version_real') and scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '5.8']) >= 0 -%>
```

Comment:
`if @collectd::collectd_version_real` should work as well. scope.lookupvar() is legacy stuff to be compatible with very old puppet versions. It isn't needed anymore

Reply:
I don't get why this is even needed, I have checked how this variable is built, and if the fact return null or undef, we pick the minimal_version. So it can never be null or I miss something ?

Reply:
I'm not 100% sure if that's still needed. That's something we could check and cleanup.